### PR TITLE
Lesson card lessons page topic pages api call and css fix

### PR DESCRIPTION
**Pull request by ilalande**
New PR getting old code of Amar's an Faycel's PR and creating a good lessonCard component which can be used for others purpuses.

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok for me

File: frontend/src/components/LessonCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import propTypes from "prop-types";
+import SLessonCard from "./style";
+import Play from "../../assets/play-solid.svg";
```
Comment (LuigiMenez) [inline frontend/src/components/LessonCard/index.jsx:3]: use the alias @

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Use @ for the routes

**Review by ThomasDCW — DISMISSED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — DISMISSED** [review]
When editing directly from GHub, it's important to indicate what you're doing.
Since you didn't do it (though I spoke about it last week), that's now forbidden -_-

**Review by ThomasDCW — APPROVED** [review]
ok for me

File: backend/src/router.js [hunk 2]
```diff
@@ -43,13 +38,13 @@ router.put("/users/:id", UserController.edit);
 router.get("/topics", TopicController.browse);
 router.get("/usertopics/:id", UserTopicController.browse);
 
-//temporary road before authentification
+// temporary road before authentification
```
Comment (LuigiMenez) [inline backend/src/router.js:41]: File again

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**